release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -91,7 +90,7 @@ Previously, {op-system} DHCP kernel parameters were not working as expected beca
 [id="ocp-4-7-rhcos-supports-multipath"]
 ==== {op-system} supports multipath
 
-{op-system} now supports multipath on the primary disk, allowing stronger resilience to hardware failure so that you can set up {op-system} on top of multipath to achieve higher host availability. See link:https://bugzilla.redhat.com/show_bug.cgi?id=1886229[*BZ#1886229*] for more information.
+{op-system} now supports multipath on the primary disk, allowing stronger resilience to hardware failure so that you can set up {op-system} on top of multipath to achieve higher host availability. Note that multipathing should only be enabled via kernel arguments within a MachineConfig as documented. Specifically, it shouldn't be enabled at installation time. See link:https://bugzilla.redhat.com/show_bug.cgi?id=1886229[*BZ#1886229*] for more information.
```

Comment:
I wanted to link to the part of the docs where we show the multipath MachineConfig, but... while I see it in the repo, I don't see that page anywhere in the live site. Are we missing an include?

Reply:
It looks like the MachineConfigs with multipaths are hidden behind an `if` statement for IBM Z systems:

https://raw.githubusercontent.com/openshift/openshift-docs/enterprise-4.7/modules/installation-complete-user-infra.adoc

```
ifdef::ibm-z,ibm-power[]
. For an installation with FCP, additional steps are required to enable multipathing. 
.. To enable multipathing on master nodes, apply the following machine config file: 
+
[source,yaml]
--
apiVersion: machineconfiguration.openshift.io/v1
kind: MachineConfig
metadata:
  labels:
    machineconfiguration.openshift.io/role: "master"
  name: 99-master-kargs-mpath
spec:
  kernelArguments:
    - 'rd.multipath=default'
    - 'root=/dev/disk/by-label/dm-mpath-root'
--

.. To enable multipathing on worker nodes, apply the following machine config file: 
+
[source,yaml]
```

Reply:
See the tail end of this section - https://docs.openshift.com/container-platform/4.7/installing/installing_ibm_power/installing-ibm-power.html#installation-complete-user-infra_installing-ibm-power

Reply:
Ahhh, good find! Oddly, doing a search for "multipath" on the docs site doesn't find that section.

So yeah, I think we should drop the `ifdef` here and have it included for not just IBM Z.

Reply:
> Ahhh, good find! Oddly, doing a search for "multipath" on the docs site doesn't find that section.

Yeah, I find the search on the site to be a bit unreliable

> So yeah, I think we should drop the `ifdef` here and have it included for not just IBM Z.

Agreed!